mmv1/third_party/terraform/tpgresource/labels.go

```diff
@@ -29,6 +29,27 @@ func SetTerraformLabelsDiff(_ context.Context, d *schema.ResourceDiff, meta inte
 	return nil
 }
 
+func SetEffectiveLabelsDiff(_ context.Context, d *schema.ResourceDiff, meta interface{}) error {
```

Comment:
What circumstances do we apply `SetTerraformLabelsDiff` but not this CustomizeDiff? As-is we'd see unexpected behaviour if someone added them in the wrong order, right? (same for other variants)

Reply:
No such circumstances. I will merge these two costomizeDiff to avoid the wrong oder issue. Thanks for pointing it out.